### PR TITLE
 charter

### PR DESCRIPTION
**Pull request by fabriziopandini**
Primo draft del community charter
Feedbacks are welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,78 @@
...
+In alternativa, puoi cercare tra le proposte esistenti, aggiungere un commento con un 
+semplice +1 o con la tua opinione in merito.
+
+### Proporre un Talk
+
+Nell'interesse della community e dei partecipanti agli eventi, distinguiamo tra talk "dalla community per la community", e sponsored talk. 
+
+Uno sponsored talk è un talk che contiene:
+- Sales or marketing pitches
+- Unlicensed or potentially closed-source technologies
```
Comment (blues-man) [inline README.md]: I'm against this, I would rather add "Do not include unlicensed or potentially closed-source technologies "

Comment (fabriziopandini) [reply README.md]: Sorry but I'm not sure I understand the suggestion:

The paragraph starts with "Uno sponsored talk è un talk che contiene:" and then you want to add "Do not include ... " in the bullet list, is that right?

**Comment by jnardiello** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (blues-man) [reply README.md]: I would remove completely line 44 to avoid closed-source technologies promotion

**Comment by blues-man** [comment]
OK also for me, I would like to understand what kind of sponsored talks are common in KCD

File: README.md (hunk 1 above)
Comment (fabriziopandini) [reply README.md]: @blues-man, comment addressed, PTAL

**Comment by fabriziopandini** [comment]
As per comments on telegram, go!

**Pull request merged**